src/commands/Fun/8ball.ts

```diff
@@ -35,9 +35,9 @@ export default class extends SkyraCommand {
 		const prefixes = cast<EightBallLanguage>(i18n.language.command8ballQuestions || this.client.languages.default.language.command8ballQuestions);
 
 		for (const key of QUESTION_KEYS) {
-			if (this.check(prefixes[key], input)) return pickRandom(i18n.get(`command8ball${key}` as ReplyTypes));
```

Comment:
Should we make a function to resolve this? `LanguageKeys.Commands.Fun.8ballResolveKey(key)` which returns one of the possible namespaces